<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmAboutFormat</class>
 <widget class="QDialog" name="FmAboutFormat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About format</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <item row="1" column="0">
      <widget class="QLabel" name="lbTypeName">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lbTypeValue">
       <property name="text">
        <string notr="true">[lbTypeValue]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbNameValue">
       <property name="text">
        <string notr="true">[Name]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbIdTypeName">
       <property name="text">
        <string>ID type:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbSoftwareName">
       <property name="text">
        <string>Software:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lbIdTypeValue">
       <property name="text">
        <string notr="true">[lbIdTypeValue]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="lbSoftwareValue">
       <property name="text">
        <string notr="true">[lbSoftwareValue]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbNameName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbCapsName">
       <property name="text">
        <string>UTranslator can:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="lbCapsValue">
       <property name="text">
        <string notr="true">[lbCapsValue]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="browAbout">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
